tests/client/amo/sagas/testIndex.js

```diff
@@ -0,0 +1,15 @@
+import { fork } from 'redux-saga/effects';
+
+import rootSagas from 'amo/sagas';
+import categoriesSaga from 'amo/sagas/categories';
+
+
+describe('amo rootSagas', () => {
+  it('should get Api from state then make API request to categories', () => {
+    const sagaGenerator = rootSagas();
+
+    assert.deepEqual(sagaGenerator.next().value, [
+      fork(categoriesSaga),
```

Comment:
Fair enough actually and I think doing `rootSagas().next()` does not throw is fine. I did that and coverage remained the same; I agree that's a better and more maintainable test. 👍